File: Cap 04/Topico 4.6/Topico 4.6.3/Operador de Negacao/nottest.pas
{*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 * 
 * Capitulo ..: 4 - Topico 4.6.3
 * 
 * Programa ..: OPERAÇÃO DE NEGAÇÃO ".NÃO. / NOT"
 *}

program TesteLogicaNot;
var
  A, B, X, C: Integer;
begin
  WriteLn('TESTE LOGICO OPERADOR: NOT');
  WriteLn;
  Write('Entre valor inteiro <A>: ');
  ReadLn(A);
  Write('Entre valor inteiro <B>: ');
  ReadLn(B);
  Write('Entre valor inteiro <X>: ');
  ReadLn(X);
  if not (X > 5) then
    C := A + B
  else
    C := A - B;
  WriteLn('O valor de C = ', C);
end.

File: Cap 04/Topico 4.6/Topico 4.6.3/Operador de Negacao/nottest.lua
--[[
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.6.3
 *
 * Programa ..: OPERAÇÃO DE NEGAÇÃO ".NÃO. / NOT"
--]]

print("TESTE LOGICO OPERADOR: NOT\n")
io.write("Entre valor inteiro <A>: ")
A = tonumber(io.read())
io.write("Entre valor inteiro <B>: ")
B = tonumber(io.read())
io.write("Entre valor inteiro <X>: ")
X = tonumber(io.read())
if not (X > 5) then
  C = A + B
else
  C = A - B
end
print("O valor de C = " .. C)


